OneCommunityGlobal/HighestGoodNetworkApp
# Peterson_Fix_Tasks_tab_revert_to_Current_Week_Timelog_tab_for_Volunte…

# Description
This pull request has been opened to fix the tasks tab reverting to the Current Week Timelog tab for the Volunteer user account, but for any other type of user, the selected tab should be tasks.

## Related PRS (if any):
None
…

## Main changes explained:
The Timelog.jsx component has been modified to fix the bug.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Log in as a volunteer user with or without tasks, or any other type of user.
5. go to dashboard
6. After the page reloads, if your volunteer user has tasks, the selected tab should be "Tasks". If your volunteer user does not have tasks, the selected tab should be "Current Week Timelog". If you test with any other type of user, the selected tab should be "Tasks".

## Screenshots or videos of changes:
![Peterson_Fix_Tasks_tab_revert_to_Current_Week_Timelog_tab_for_Volunteer_user_accoun](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73541319/454785c3-f868-48ef-ace1-92e02c0c752e)

## Note:
None.
